build-files/prokka/1.15.6/Dockerfile

```diff
@@ -0,0 +1,78 @@
+
+ARG PROKKA_VER="1.15.6"
+ARG MINCED_VER="0.4.2"
+
+FROM ubuntu:noble AS app
+
+ARG PROKKA_VER
+ARG MINCED_VER
+
+LABEL base.image="ubuntu:noble"
+LABEL dockerfile.version="1"
+LABEL software="Prokka"
+LABEL software.version="${PROKKA_VER}"
+LABEL description="Automated prokaryotic genome annotation tool"
+LABEL website="https://github.com/tseemann/prokka"
+LABEL license="https://github.com/tseemann/prokka#licence"
+LABEL maintainer="Curtis Kapsak"
+LABEL maintainer.email="kapsakcj@gmail.com"
+LABEL maintainer2="Kutluhan Incekara"
+LABEL maintainer2.email="kutluhan.incekara@ct.gov"
+
+ARG DEBIAN_FRONTEND="noninteractive"
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    wget \
+    ca-certificates \
+    perl \
+    aragorn \
+    barrnap \
+    ncbi-blast+ \
+    hmmer \
+    infernal \
+    parallel \
+    bioperl \
+    libbio-searchio-hmmer-perl \
+    libxml-simple-perl \
+    prodigal \
+    ncbi-tools-bin \
+    default-jre-headless \
+    dpkg-dev \
+    gcc
```

Comment:
This doesn't clean up the extra apt-get files. What happens when you add something like the following to the apt-get layer?

```
rm -rf /var/lib/apt/lists/* && apt-get autoclean
```